package.json

```diff
@@ -39,6 +39,7 @@
     "semver": "^5.2.0",
     "touch": "^1.0.0",
     "update-json": "^1.0.0",
-    "yargs": "^4.7.1"
+    "yargs": "^4.7.1",
+    "replace-in-file": "^2.5.0"
```

Comment:
Could you add this with `npm install --save replace-in-file` please? That way it will be added alphabetically. Alternatively you can just move it into the correct position yourself, but I thought I should let you know about `npm i --save` :)